site/docs/install.md

```diff
@@ -5,14 +5,17 @@ title: Installing Bazel
 
 # Installing Bazel
 
-See the instructions for installing Bazel on:
+The Bazel team officially supports Bazel on the following platforms. See the instructions for installing Bazel on:
 
 *   [Ubuntu Linux](install-ubuntu.md)
 *   [macOS](install-os-x.md)
 *   [Windows](install-windows.md)
 
 ## Community-supported packages
 
+These packages are maintained by Bazel community members and are not officially supported by the Bazel Team. Please 
+reach out to the respective package maintainers for support and questions.
```

Comment:
Non-passive voice idea: "Bazel community members maintain these packages. The Bazel team does not officially support them" (this would also be consistent with the wording in the previous section).

Reply:
Done.